src/main/java/com/amazon/ion/system/IonSystemBuilder.java

```diff
@@ -271,7 +276,111 @@ public final IonSystemBuilder withStreamCopyOptimized(boolean optimized)
         return b;
     }
 
+    //=========================================================================
+
+    /**
+     * Gets the text writer builder whose options will be used when building an
+     * {@link IonSystem}. By default, {@link IonTextWriterBuilder#standard()} will
+     * be used.
```

Comment:
This may be surprising to users who want to add their own configuration to the default text writer builder.
```java
IonSystemBuilder isb = IonSystemBuilder.standard();
isb.getIonTextWriterBuilder()
    .setNewLineType(NewLineType.LF); // NullPointerException!
IonSystem ionSystem = isb.build();
```
If we initialize `textWriterBuilder` to the default value, then we could ensure that `getIonTextWriterBuilder()` is always non-null.